doc/en/user/source/introduction/gettinginvolved.rst

```diff
@@ -1,52 +1,66 @@
 
 Documentation
 -------------
 
-One of the best and most needed ways to help out is with the documentation. Our official documentation is contained as part of our official code repository in order to maintain a uniform look and feel.
+Another crucial way to help out is with documentation. Whether it's adding tutorials or just correcting mistakes, every contribution serves to make the project more healthy. And the best part is that you do not need to be a developer in order to contribute.
+
+Our official documentation is contained as part of our `official code repository <https://www.github.com/geoserver/>`_. As part of the GitHub model, anyone can submit patches as pull requests, which will be evaluated by the team. To learn more about contributing to the GeoServer codebase, we highly recommend joining the GeoServer developers mailing list. See details below.
 
 Mailing lists
 -------------
 
-GeoServer maintains two email lists: `GeoServer Users <http://lists.sourceforge.net/lists/listinfo/geoserver-users>`_ and `GeoServer Developers <http://lists.sourceforge.net/lists/listinfo/geoserver-devel>`_. These lists are publicly available and are a great resource for those who are new to GeoServer, who need a question answered, or who are interested in contributing code. The Users list is mainly for those who have questions relating to the use of GeoServer, and the Developers list is for more code-specific and roadmap-based discussions. If you see a question asked on these lists that you know the answer to, please respond! 
+GeoServer maintains two email lists:
+
+* `GeoServer Users <http://lists.sourceforge.net/lists/listinfo/geoserver-users>`_
+* `GeoServer Developers <http://lists.sourceforge.net/lists/listinfo/geoserver-devel>`_
+
+The Users list is mainly for those who have questions relating to the use of GeoServer, and the Developers list is for more code-specific and roadmap-based discussions. If you see a question asked on these lists that you know the answer to, please respond!
+
+These lists are publicly available and are a great resource for those who are new to GeoServer, who need a question answered, or who are interested in contributing code. 
 
 IRC
 ---
 
-GeoServer has an IRC channel, `#geoserver <irc://irc.freenode.net/geoserver>`_, on the `Freenode <http://freenode.net>`_ network. GeoServer developers frequent this channel, and so it is a great way to give and receive information in real time. 
+GeoServer has an IRC channel, `#geoserver <irc://irc.freenode.net/geoserver>`_, on the `Freenode <http://freenode.net>`_ network. GeoServer developers frequent this channel, and so it is a great way to give and receive information in real time.
```

Comment:
Ask geoserver-devel about retiring this (or replacing with stack exchange). IRC is no longer watched actively.